app/components/windows/go-live/CommonPlatformFields.tsx

```diff
@@ -176,7 +176,7 @@ export default class CommonPlatformFields extends TsxComponent<Props> {
             <div>
               {/*TITLE*/}
               <HFormGroup
-                value={fields.title}
+                value={fields['title']}
```

Comment:
not blocking but wondering why this change was made

Reply:
There is a TS type issue that I'm not sure how to resolve with massive changes in type description. `fields.title` used to be a field for all platforms but it's not true now. However, we never show `CommonPlatformFields.tsx` for TikTok so should be no real errors here